### PR TITLE
DEMO-144: [User Profiling block] unable to fetch autocomplete for tags

### PR DESCRIPTION
**Pull request by damianz5**
https://jira.ez.no/browse/DEMO-144

### CODE REVIEW

File: src/AppBundle/Resources/public/js/views/up-itemsfieldview.js [hunk 1]
```diff
@@ -132,7 +132,7 @@ YUI.add('up-itemsfieldview', function (Y) {
 
             if (selector) {
                 jQuery(selector).EzTags({
-                    autocompleteUrl: '/tags/admin/field/autocomplete?',
+                    autocompleteUrl: '/admin/tags/admin/field/autocomplete?',
```
Comment (SylvainGuittard) [inline src/AppBundle/Resources/public/js/views/up-itemsfieldview.js]: Not a big fan of this solution...
What if the admin siteaccess is different?

Comment (damianz5) [reply src/AppBundle/Resources/public/js/views/up-itemsfieldview.js]: Right, I'm looking for better solution

Comment (webhdx) [reply src/AppBundle/Resources/public/js/views/up-itemsfieldview.js]: Please take a look at: https://symfony.com/doc/master/bundles/FOSJsRoutingBundle/usage.html

**Comment by SylvainGuittard** [comment]
Definitely safer :)

**Pull request merged**